models/migrations/migrations.go

```diff
@@ -100,6 +100,8 @@ var migrations = []Migration{
 	NewMigration("change the key_id and primary_key_id type", changeGPGKeysColumns),
 	// v25 -> v26
 	NewMigration("add show field in user openid table", addUserOpenIDShow),
+  // v26 -> v27
+  NewMigration("generate and migrate repo and wiki Git hooks", generateAndMigrateGitHookChains),
```

Comment:
Run `make fmt` should be enough (and we need our bot to guard after this, I may give that a try)